src/sentry/static/sentry/app/views/events/utils/getParams.tsx

```diff
@@ -39,7 +48,7 @@ export function getParams(params = {}) {
     .reduce(
       (acc, [key, value]) => ({
         ...acc,
-        [key]: value,
+        [key]: value as string,
```

Comment:
Should you use a string cast instead of a compiler hint?

Reply:
Turns out `value` can also be `string[]` 😄